src/param.rs

```diff
@@ -112,6 +116,10 @@ pub enum Param {
     /// For Groups and Contacts
     ProfileImage = b'i',
 
+    /// For contacts, contains the chat id of the mailing list if this is a pseudo contacts.
+    /// If the List-Id is e.g. delta.chat.1234, then the pseudo contact is delta.chat.1234@mailing.list
+    MailingListPseudoContact = b'p',
```

Comment:
let's see if i understand correctly :)

so, `MailingListPseudoContact` is _not_ set for the concrete senders?

if 3 different senders all use the same email-address but different names (as of `Alice <notifications@github.com>`, `Bob <notifications@github.com>`, `Claire <notifications@github.com>`):
- that would result in only one contact-db-entry with the address  `notifications@github.com`, MailingListPseudoContact would _not_ bet set?
- the three messages would get `OverrideSenderDisplayname` set to `Alice`, `Bob` and `Claire`?

Reply:
Both correct, maybe this needs another comment if it's not clear, let's not mark this conversation as resolved until i did this.

Yes, _All_ mailing list pseudo contacts end on `@mailing.list` and are made up by DC so the user can unblock mailing lists.

Reply:
Maybe make a proper way to unblock groups instead?

Reply:
What do you have in mind for unblocking groups? Rename "Blocked contacts" to "Blocked contacts and groups" (if we also mention mailing lists, this is going to get too long for a settings entry, but maybe we come up with a better name) and show groups in the same way as we do in the title bar (e.g. group name in big, members number in small)

Reply:
> Rename "Blocked contacts" to "Blocked contacts and groups" (if we also mention mailing lists, this is going to get too long for a settings entry, but maybe we come up with a better name) and show groups in the same way as we do in the title bar (e.g. group name in big, members number in small)

Rename it to "blocked chats" and show 1:1 chats for blocked contacts and group chats/mailing list chats for blocked groups. For 1:1 chats ignore the "blocked" column.

What happens when blocked contact adds you to a group? What happens if someone adds you to a group with a blocked contact?